cves/kernel/CVE-2016-2782.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: "If an attacked used a broken or intentionally modified USB device without a bulk-in/interrupt-in endpoint, it could cause the kernal to completely crash and require a reboot."
```

Comment:
👍🏾, just fix the typos (attacked and kernal) and state what kind of attack is caused by this